src/Components/BookPage/BookCover.js

```diff
@@ -11,11 +11,15 @@ const styles = {
   bookCover: {
     maxWidth: '20vw',
     filter: 'drop-shadow(4px 4px 50px rgba(0, 0, 0, 0.25))',
+
     marginRight: '2vw',
     '@media (max-width: 960px)': {
-      maxWidth: '90vw',
+      maxWidth: '75vw',
       margin: 'auto',
+      filter: 'drop-shadow(0px 4px 6px rgba(0, 0, 0, 0.25))',
+      borderRadius: '16px',
     },
+
   },
   topDownContainer: {
```

Comment:
The auto margins on the topDownContainer cause the cover + tags to be centered rather than start justified. I think it might be nice to change that as the other 2 sections are justified at the start + it makes a difference at smaller widths, unless Chantal has already told you otherwise!